asab/storage/elasticsearch.py

```diff
@@ -722,10 +730,7 @@ def __init__(self, storage, collection, obj_id, version=None):
 		if version == 0:
 			self.ModSet['_c'] = now  # Set the creation timestamp
 
-		api_key = Config.get('asab:storage', 'elasticsearch_api_key')
-		self.Headers = {'Content-Type': 'application/json'}
-		if api_key != '':
-			self.Headers['Authorization'] = "ApiKey {}".format(api_key)
+		self.Headers = self.Storage.Headers
```

Comment:
Why?

Reply:
Upsertor is a transient object (it has a short lifecycle), use `self.Storage.Headers` directly.

Reply:
Copied from PR #513